protected/views/patient/crud/_form.php

```diff
@@ -561,10 +561,10 @@
 
 <script type="text/javascript">
   function findDuplicates(id) {
-    if ($("#fname").val() && $('#surname').val() && $('#Patient_dob').val()) {
+    if ($("#Contact_first_name").val() && $('#Contact_last_name').val() && $('#Patient_dob').val()) {
```

Comment:
We could probably use single quotes here for #Contact_first_name. I'm surprised PHPStorm didn't pick this up?

Reply:
Good catch